akka-http/src/main/scala/akka/http/javadsl/marshalling/Marshaller.scala

```diff
@@ -88,8 +88,8 @@ object Marshaller {
    * Helper for creating a "super-marshaller" from a number of "sub-marshallers".
    * Content-negotiation determines, which "sub-marshaller" eventually gets to do the job.
    */
-  def oneOf[A, B](ms: Marshaller[A, B]*): Marshaller[A, B] = {
-    fromScala(marshalling.Marshaller.oneOf[A, B](ms.map(_.asScala): _*))
+  def oneOf[A, B](m1: Marshaller[A, B], m2: Marshaller[A, B]): Marshaller[A, B] = {
```

Comment:
It didn't work, compiler/interop issue, see the existing comment above. I think the other n-arity methods were introduced for the same reason, and this line was perhaps simply forgotten?